src/dark-mode-toggle.mjs

```diff
@@ -236,8 +236,6 @@ aside {
 </style>
 <form>
 <fieldset>
-  <legend></legend>
```

Comment:
Perfectly valid to omit the `<legend>`, and it especially makes sense if it's just empty anyhow.

Reply:
It [will have a value](https://github.com/GoogleChromeLabs/dark-mode-toggle/blob/be335afdbf37201887ce7c5aec163cb0d9682a94/src/dark-mode-toggle.mjs#L424) once the user sets the [`legend="Dark Mode Toggle"` attribute](https://github.com/GoogleChromeLabs/dark-mode-toggle#properties).